src/Model/User.php

```diff
@@ -127,18 +127,18 @@ public static function getIdFromPasswordAuthentication($user_info, $password)
 	{
 		$user = self::getAuthenticationInfo($user_info);
 
-		if ($user['legacy_password']) {
```

Comment:
Can you please restore these lines and replace the original condition with the new strpos? Thanks!

Reply:
You mean to swap the `if` and `elseif` conditions?

Reply:
Yes, to preserve a single password_verify call by execution.

Reply:
Here you are https://github.com/friendica/friendica/pull/4782/commits/e860cdf6a8769c8441ed3dba33f192898c78ab40